spec/models/inventory_spec.rb

```diff
@@ -52,11 +52,13 @@
   end
 
   describe '#save' do
-    subject {
-      create(:inventory)
-    }
+    let(:owner) { FactoryGirl.create(:user) }
+    subject { FactoryGirl.create(:inventory, owner: owner) }
```

Comment:
the factory should be calling `save`, if it calls `save` we'll get `after_create` calling `add_facilitator_owner` which is the default behavior for factories and non-factories.

I'm not sure what your point is.